projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -6,22 +6,31 @@
 .ng-select .ng-select-container {
   @include regular-input;
 
-  input {
-    font-size: 1rem;
-    font-weight: 300;
+  .ng-value-container .ng-input>input {
```

Comment:
I'm used to selectors having spacing around them, like `.ng-input > input`. Do we want those spaces here?

Reply:
Nah. fixed it.